pages/event-types/[type].tsx

```diff
@@ -1070,19 +1060,6 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                 <LinkIcon className="w-4 h-4 mr-2 text-neutral-500" />
                 {t("copy_link")}
               </button>
-              <Dialog>
```

Comment:
Instead of deleting this Dialog, can you dynamically show the `<Dialog>` if the `user.email` is in `process.env.THETIS_ADMIN_USER_EMAILS` ? Something like this:

```
const isAdminUser = process.env.THETIS_ADMIN_USER_EMAILS?.split(";").includes(user.email);`

{isAdminUser && <Dialog> .....} 
```

This allows us to eventually re-enable this feature for Admin users to debug issues if they need to impersonate the instructors account and login as them. My bad for not clarifying what I meant by "Hide" in the CU spec!